src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesArray = stylesString.split(';');
+  const stylesObject = {};
+
+  for (let i = 0; i < stylesArray.length; i++) {
+    const parts = stylesArray[i].split(':');
+
+    if (parts[0] && parts[1]) {
+      const property = parts[0].trim();
+      let value = parts[1].trim();
+
+      if (value.includes('!important')) {
+        value = value.replace(/\s*!important\s*/, '') + ' !important';
+      }
+
+      stylesObject[property] = value;
+    }
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. This can be achieved by using `forEach`, `map`, `filter`, etc. For example, you can use `map` to split each style declaration, then `filter` to remove any empty ones, and finally reduce to accumulate them into an object.